src/content/en/2022/markup.md

```diff
@@ -1,22 +1,555 @@
+        <td>`link`</td>
+        <td>`link`</td>
+        <td>`link`</td>
+      </tr>
+      <tr>
+        <td></td>
+        <td>`i`</td>
+        <td>`meta`</td>
+        <td>`i`</td>
+      </tr>
+      <tr>
+        <td></td>
+        <td>`option`</td>
+        <td>`i`</td>
+        <td>`meta`</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>
+    {{ figure_link(
+      caption="Most used elements.",
+      sheets_gid="571472591",
+      sql_file="element_frequency.sql",
+    ) }}
+  </figcaption>
+</figure>
+
+The `div` element is—by far—the most popular element: We found 2,123,819,193 occurrences in the mobile data set, and 1,522,017,185 of them in our desktop data set.
+
+{{ figure_markup(
+  content="29%",
+  caption="Percentage of elements which are `div` elements.",
+  classes="big-number",
+  sheets_gid="571472591",
+  sql_file="element_frequency.sql",
+) }}
+
+[Divitis](https://en.wiktionary.org/wiki/divitis) is real.
+
+If you wonder about the odd one out, [the `i` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i), it stands to reason that this is still largely due to <a hreflang="en" href="https://fontawesome.com/">Font Awesome</a> and its arguable misuse of this element. The element has also a bad reputation because during XHTML times, everyone suggested to use `em` instead—but that advice wasn't sound, and <a hreflang="en" href="https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element">`i` elements have their use cases</a>.
+
+When it comes to what elements are being used on the most documents, the list looks a little different:
+
+{{ figure_markup(
+  image="adoption-of-top-html-elements.png",
+  caption="Adoption of top HTML elements.",
+  description="Bar bar chart showing `html` is used on 99.3% of desktop  and 99.4% of mobile page, `head` on 99.3% and 99.4% respectively, `body` on 99.1% and 99.2%, `title` on 98.9% and 99.0%, `meta` on 98.5% and 98.9%, `div` on 98.3% and 98.5%, `a` on 98.0% and 98.1%, `link` on 97.8% and 98.0%, `script` on 97.6% and 97.8%, `img` on 95.9% and 96.1%, `span` on 94.2% and 94.7%, `p` on 89.9% and 90.0%, `ul` on 88.8% and 88.7%, and finally `li` on 88.7% of desktop and 88.6% of mobile pages.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRjXjOczLu9q3mcQ-UFLUOmZgefGNCPVYeEwo4cDxQTymgmD_1D5dbZ728Mz1SkEMZHxQgwcWmLjkgx/pubchart?oid=714125011&format=interactive",
+  sheets_gid="2057119066",
+  sql_file="element_popularity.sql",
+) }}
+
+But here it's not a surprise that not every document uses `html`, `head`, or `body` tags—the respective tags are optional in HTML and are automatically inserted in the DOM, which is what is being counted here. The reason these are slightly less than 100%, are due to the a small number of pages breaking our detection by overriding JavaScript APIs we use for this detection—for example <a hreflang="en" href="https://mootools.net/">MooTools</a> overriding the `JSON.stringify()` API we use.
```

Comment:
```suggestion
But here it's not a surprise that not every document uses `html`, `head`, or `body` tags—but not because the respective tags are optional in HTML. As they are automatically inserted in the DOM, they are being counted here. That the numbers are slightly less than 100% is due to a small number of pages that break detection by overriding the JavaScript APIs we use—for example, <a hreflang="en" href="https://mootools.net/">MooTools</a> overriding the `JSON.stringify()` API.
```

Reply:
Got another suggestion above. Let me know what you think.

Reply:
Can we replace “But here it’s” by “It’s”, and in the following paragraph, “It’s a lot more surprising” by “But it is a surprise”? I think that would read better then.

Optional: How about breaking the first sentence up, perhaps by a period after “tags”, and then continuing with “They are…”?

And, what do you think of the double “overriding”? Missed this earlier, but caught it now.

Reply:
> Can we replace “But here it’s” by “It’s”, and in the following paragraph, “It’s a lot more surprising” by “But it is a surprise”? I think that would read better then.

Did the first.

Not sure the second is a great idea. Sentences beginning with "But" is generally bad form—it's an indication of a continuation of the previous thought so should really be in the same paragtpah—and for paragraphs even more so!

> Optional: How about breaking the first sentence up, perhaps by a period after “tags”, and then continuing with “They are…”?

I wouldn't break it as seems like the reason it's not a surprise is at least in part because the browser is automatically inserting them. I replaced with an em-dash instead in above suggestion.

I think the double overriding is fine here. We're saying "it's caused by overrides, such as this example override" so it's a deliberate repeat.

Reply:
I think it reads better, thanks.

The suggestion to start the next paragraph with “But it is…” was deliberate, for emphasis. (“It is not a surprise that… But it is a surprise that…”.) This is not to argue about it; let’s defer this point and look at how the final/live version reads. Does that work for you?